### DISCUSSION TITLE
Material Adicional

### DISCUSSION THREAD

**Discussion by 4klayton** [balta-io/forum#278] [Tira Dúvidas] [answered]
Alguns vídeos, principalmente os primeiros passos do curso de fundamentos o André menciona que vai deixar um material adicional para algumas tarefas auxiliares ao treinamento. O problema é que eu não sei onde fica esse material. 
Só encontrei 2 links na página dos vídeos, um leva para esse fórum e o segundo para o repositório do curso no github e lá eu só encontro o PPT da aula e os tópicos do material adicional num arquivo MarkDown. 

Onde estou errando?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @4klayton, como vai?
São estes materiais mesmo... além deles tem nosso grupo no Discord (https://balta.io/discord), sempre fazemos desafios lá.
